test/integration-test-v2.py

```diff
@@ -315,6 +319,41 @@ def test_sct_embedding():
         if sct.entry_type != x509.certificate_transparency.LogEntryType.PRE_CERTIFICATE:
             raise Exception("SCT contains wrong entry type")
 
+def uninitialized_client(key=None):
```

Comment:
Let's make uninitialized_client part of chisel2 and call it as part of make_client.